README.md

```diff
@@ -48,7 +48,10 @@ No. We continue to consume Fluent Bit from its main repository. We are not forki
 
 ### Public Images
 
-Each release updates the `latest` tag and adds a tag for the version of the image.
+Each release updates the `latest` tag and adds a tag for the version of the image. A `stable` tag is also available which can be trusted that it is the latest version in which there are no high impact bugs. A release may be marked as stable if the following rules are all met:
+* It has been out for a period of time
+* The Fluent Bit version contained in the release is a patch version as the patch version indicates the release includes bug fixes
+* It has been no bugs reported which will have high impact for any of our customers
 
```

Comment:
Another idea is to create a subsection with title like "Using stable tag to find the latest version". But I think it may be a little inconsistent with "Using SSM to find available versions". Not sure which way is better. Any suggestions?

Reply:
I think actually it'd be best to create section right here, before "Using SSM to find available versions" which is titled

"Using the stable tag" or "Stable release tag". something like that.

That way it jumps out at you and is in the outline at the top. Otherwise it's very easy to miss this information.